### ISSUE TITLE
Problems with manual relocation of judges

### ISSUE THREAD

**Issue by aosterkorn** [Tabbie2/issues#3]
At ZEIT DEBATTE Göttingen the feature to move judges between rooms before publishing of the round was used rather intensely.
After having dragged and dropped the judges and publishing of the tab there was a deviation to the manually adjusted allocation on the presented draw and on the printed ballots.
We only saw cases in which the chair was assigned wrongly but all judges were in the correct adjusted rooms. (see e. g. ZEIT DEBATTE Göttingen, round #4, room 27 Purgatorio)

I can`t give a pattern for this chair reallocation but that part should definitely be reviewed.

**Comment by chief-nerd** [comment]
I'll have a look

**Comment by rscoates** [comment]
This also came up at Cambridge Womens

**Comment by Lord-Bean** [comment]
Had this happen to me at Erasmus Rotterdam Open this weekend. When drawing the first round using preset panels it first dropped every panel in every room (I commented on the topic for that seperately as well). 

I immediately removed all preset panels and redrew the round and manually dragged and dropped untill I had the desired panels again. I published the draw and proceded to print the ballots, only to notice then that the panels were all messed up. Couldn't change it anymore since some chairs were even left with no wings and I could only switch them. 

The weird thing was furthermore that the redistributed panels were signficantly worse by any standards, with five of them having institutional clashes (even a lone chair one), as well as two high ranked chairs winging to lower ranked adjudicators marked as non-chairs.
